src/meta/api/src/schema_api.rs

```diff
@@ -243,7 +243,7 @@ pub trait SchemaApi: Send + Sync {
     async fn update_multi_table_meta(
         &self,
         req: UpdateMultiTableMetaReq,
-    ) -> Result<Vec<u64>, KVAppError>;
+    ) -> Result<Vec<(u64, u64, TableMeta)>, KVAppError>;
```

Comment:
I'd suggest changing the return value to `Result<Result<(), Vec<_>>, KVAppError>`.

This change ensures that an `Ok` return value explicitly represents a **partial** success, which could be dangerous if overlooked by a developer. By using a nested `Result`, we make the partial success unmistakable, as `Result` is a must-use value that generates a compiler warning if not handled. This approach enhances the code's safety by enforcing error handling.